Assets/ScrollSnap.cs

```diff
@@ -195,7 +195,7 @@ public class OnReleaseEvent : UnityEvent<int> {}
 	}
 
 	void LerpToElement() {
-		float t = (float)((DateTime.Now - lerpStartedAt).TotalMilliseconds / lerpTimeMilliSeconds);
+		float t = (float)((Time.time - lerpStartedAt) * 1000 / lerpTimeMilliSeconds);
```

Comment:
Now that all the values here are float, can you change it to the following 

```cs
float t = (Time.time - lerpStartedAt) * 1000f / lerpTimeMilliSeconds;
```

Reply:
Done :)